CHANGELOG.md.20190330

```diff
@@ -0,0 +1 @@
+find . -iname 'dockerfile' -exec sed -i "s/FROM.*/ARG DISTRO=alpine:3.8\nFROM \$DISTRO/" '{}' \;
```

Comment:
Sorry, I might have missed this one before, but I believe this file does not belong here?

Reply:
Functional testing went fine :+1:  So ready to approve if this little piggie is clarified.

Reply:
Yep, my bad, I should not have commited this. Thanks !